### DISCUSSION TITLE
&#102;这种格式的Unicode解码在哪里

### DISCUSSION THREAD

**Discussion by Asura88** [he3-app/he3-feedback#223] [General]
&#102;这种格式的Unicode解码在哪里
![image](https://github.com/he3-app/he3-feedback/assets/25000885/641b2f31-a1d5-40e3-a89b-21d773cff4cf)

**Comment by lyzhang1999** [comment]
感谢反馈，目前暂时没有该功能，你可以关注这个 issue：https://github.com/he3-app/he3-feedback/issues/224
